src/main/java/com/userservice/service/UserServiceImpl.java

```diff
@@ -48,35 +54,41 @@ public User createUser(User user) throws UserExistsException {
      * @return User
      */
     @Override
-    public User findUserById(String id) {
-        log.info("Entering UserServiceImpl.findUserById with parameter id {}.", id);
+    public User findUserById(String id, String loggedInUserId) {
+        log.info("Entering UserServiceImpl.findUserById with parameter id {}", id);
```

Comment:
In all the logging of this class, print the loggedInUserId as well.

Reply:
While User creation there is no authentication except that method i am logging loggedInUserId.